Resources/Prototypes/Entities/Structures/Storage/Crates/crates.yml

```diff
@@ -261,7 +261,7 @@
   components:
   - type: AccessReader
     access: [["Security"]]
-  - type: SecureEntityStorage
+  - type: Lock
```

Comment:
same as above etc etc

Reply:
do these prototypes really not have a common parent? you dont have to change it in this PR if you dont want but it seems like this is a lot of unnecessary duplication

Reply:
I'll do it in the EntityStorage ECS refactor